index.js

```diff
@@ -32,6 +37,51 @@ module.exports = {
         ],
       },
     },
+    delete: {
+      validation: {
+        function: 'checkboxValidation',
+        on: 'change',
+      },
+      script: {
+        function: 'fileScript',
+        on: 'change',
+      },
+      type: 'checkbox',
+      label: 'Delete file',
+      options: [
+        { label: 'Delete file',
+          value: 'true',
+        },
+      ],
+    },
   },
-  html: '<label for="{{file.id}}">{{file.label}}</label><input type="{{file.type}}" id="{{file.id}}" name="{{file.name}}" value="{{file.value}}" {% if settings.types %}{%set comma = joiner() %}accept="{% for type in settings.type %}{{comma()}}{{type}}{% endfor %}{% endif %}" />',
+  html: `<label for="{{file.id}}">{{file.label}}</label>
+    <div class="field--upload">
```

Comment:
Why is this wrapped in a div? And we're going to want the `field--upload` class on the `input`